### ISSUE TITLE
Payload missing should be handled 

### ISSUE THREAD

**Issue by Carreau** [jupyter/jupyter_console#9]
See https://github.com/JuliaLang/IJulia.jl/pull/325#issuecomment-117330963 where `ipython console --kernel julia crashes`

cf https://github.com/jupyter/qtconsole/issues/21 also

**Comment by stevengj** [comment]
Note that I was using IPython 3.0.0, so it would be good to check whether this has already been fixed.

**Comment by Carreau** [comment]
Well you should at least update to 3.2 that fix some XSS vulnerability on notebook...

**Comment by stevengj** [comment]
Okay, looks like Anaconda has 3.2, so I will try updating...

**Comment by stevengj** [comment]
Got the same error in IPython 3.2.0:
```
  File "/Users/stevenj/anaconda/lib/python2.7/site-packages/IPython/terminal/console/interactiveshell.py", line 208, in handle_execute_reply
    for item in content["payload"]:
KeyError: 'payload'
```

**Comment by stevengj** [comment]
Looks like the unqualified check for a `payload` is still [in git master](https://github.com/jupyter/jupyter_console/blob/33e0ecb40a33a665c686ad6b1dac13e7be676ec2/jupyter_console/interactiveshell.py#L208).

**Comment by minrk** [comment]
Fixed by #10
